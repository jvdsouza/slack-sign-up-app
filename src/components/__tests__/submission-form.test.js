import React from 'react';
import renderer from 'react-test-renderer';

//component being tested
import SubmissionForm from '../submission-form';

describe("submission form", () => {
    it("renders correctly", () => {
        const tree = renderer
            .create(<SubmissionForm/>)
            .toJSON()
        expect(tree).toMatchSnapshot()
    });
});