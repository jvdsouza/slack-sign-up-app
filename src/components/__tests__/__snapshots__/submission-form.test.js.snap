// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`submission form renders correctly 1`] = `
<div>
  <div>
    <div
      className="ma2"
    >
      First Name
    </div>
    <input
      name="firstName"
      onChange={[Function]}
      type="text"
      value=""
    />
  </div>
  <div>
    <div
      className="ma2"
    >
      Last name
    </div>
    <input
      name="lastName"
      onChange={[Function]}
      type="text"
      value=""
    />
  </div>
  <div>
    <div
      className="ma2"
    >
      Email*
    </div>
    <input
      name="email"
      onChange={[Function]}
      type="text"
      value=""
    />
  </div>
  <p
    className="ma1"
  >
    * = Required
  </p>
  <button
    className="ma3 f6 link dim ba ph3 pv2 mb2 dib black arial"
    onClick={[Function]}
    type="submit"
  >
    Sign Up
  </button>
</div>
`;
